docs/troubleshooting-guide/cant-see-tables.md

```diff
@@ -0,0 +1,85 @@
+# I can't see my tables
+
+You have connected Metabase to a database, but:
+
+- you don't see the tables in the Data Model section of the Admin Panel,
+- the tables don't appear in the Data Browser,
+- the tables don't show up as possible data sources when you create a query using the Notebook Editor, or
+- you can no longer see tables that you used to be able to see.
```

Comment:
We should link to resources on data model, data browser, as we shouldn't assume readers are familiar with these, especially if they've just set up Metabase.

Reply:
:+1: added